client/my-sites/site-settings/disconnect-site/skip-survey.jsx

```diff
@@ -0,0 +1,42 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'gridicons';
+import Main from 'components/main';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+class SkipSurvey extends Component {
+	handleClick = () => {
+		const { siteSlug } = this.props;
+
+		//placeholder for redirection to the Confirm Disconnection page
+		if ( siteSlug ) {
+			page( '/settings/general/' + siteSlug );
```

Comment:
I know this will be updated, but `/settings/manage-connection` is probably less confusing for now.